admin/src/core/services/cognito.service.ts

```diff
@@ -167,10 +132,90 @@ export class CognitoService {
     return user;
   }
 
-  public getToken(): any {
+  public getToken() {
     if (!this.awsCognitoConfig.enabled) {
       return JSON.stringify(this.fakeUser);
     }
     return this.cognitoAuthToken;
   }
+
+  private async loadRemoteConfig() {
```

Comment:
No big change, method location moved.
Only need to adjust config to fit Amplify v6